server/src/controllers/Events/resolver.ts

```diff
@@ -127,16 +140,29 @@ export class EventResolver {
       if (!rsvp.on_waitlist) {
         const waitingList = event.rsvps.filter((r) => r.on_waitlist);
 
-        if (waitingList.length > 0) {
+        if (waitingList) {
```

Comment:
I don't think we can change this, because even if we filter out all the rsvps `waitingList = []` and `[]` is truthy.

Reply:
Uh, I think I confused it with just `if (waitingList.length) {`.